### DISCUSSION TITLE
Is it work ?

### DISCUSSION THREAD

**Discussion by Barladulos** [nfcgate/nfcgate#155] [Q&A]
Hello guys, i want to know if it work cause when i want to do a payment for test i m rejecting , i think is from server or protobuff.
Thanks for all!

**Comment by kleest** [comment]
We are not aware of such problems. Some people have successfully used NFCGate with payment protocols and applications. Of course there could be compatibility issues with some Android devices.

Unfortunately, your question is a bit vague. Without further details about the problem, your setup, your device, logs, etc. we cannot assist you, sorry.
